### PR TITLE
adds pen in tablet flavor text + ctrl click

### PR DESCRIPTION
**Pull request by azrael-x86**
## About The Pull Request

people seem to be confused about the fact that pens are actually inside of your tablet because of extremely awful horrible UX, so flavor text has been added

and also ctrl click to remove pens is back

## Why It's Good For The Game
my ux

## Changelog

:cl:
fix: brings back ctrl-click to remove pen + flavor text
/:cl:

### CODE REVIEW

**Comment by Coolguysixtyfour** [comment]
woah!

**Review by stylemistake — APPROVED** [review]

**Comment by cacogen** [comment]
you used to be able to ctrl+shift click to remove a pda's cartridge (now floppy disk) too

File: code/modules/modular_computers/computers/item/tablet.dm [hunk 1]
```diff
@@ -39,6 +39,11 @@
 		explode(usr, from_message_menu = TRUE)
 		return
 
+/obj/item/modular_computer/tablet/examine(mob/user)
+	. = ..()
+	if(inserted_item)
+		. += "Ctrl-click to remove the [inserted_item.name]."
```
Comment (tralezab) [inline code/modules/modular_computers/computers/item/tablet.dm:45]: Use a context screen tip instead, please! Here's an example of one, similar but not exactly what you would need:

```js
	AddElement( \
		/datum/element/contextual_screentip_bare_hands, \
		lmb_text = "Turn on", \
		rmb_text = "Turn off", \
	)
```

**Review by tralezab — CHANGES_REQUESTED** [review]
Context screen tips, hell yeah

File: code/modules/modular_computers/computers/item/tablet.dm (hunk 1 above)
Comment (stylemistake) [reply code/modules/modular_computers/computers/item/tablet.dm:45]: I don't think you actually need backslashes here.

**Review by tralezab — APPROVED** [review]
awesome work

File: code/modules/modular_computers/computers/item/computer.dm [hunk 2]
```diff
@@ -282,6 +283,14 @@ GLOBAL_LIST_EMPTY(TabletMessengers) // a list of all active messengers, similar
 
 	. += get_modular_computer_parts_examine(user)
 
+/obj/item/modular_computer/add_context(atom/source, list/context, obj/item/held_item, mob/user)
+	. = ..()
+
+	context[SCREENTIP_CONTEXT_ALT_LMB] = "Remove ID"
+	context[SCREENTIP_CONTEXT_CTRL_SHIFT_LMB] = "Remove Job Disk"
+
+	. = CONTEXTUAL_SCREENTIP_SET
```
Comment (Mothblocks) [inline code/modules/modular_computers/computers/item/computer.dm:292]:
```suggestion
	return CONTEXTUAL_SCREENTIP_SET
```

File: code/modules/modular_computers/computers/item/tablet.dm [hunk 3]
```diff
@@ -39,6 +39,13 @@
 		explode(usr, from_message_menu = TRUE)
 		return
 
+/obj/item/modular_computer/tablet/add_context(atom/source, list/context, obj/item/held_item, mob/user)
+	. = ..()
+
+	context[SCREENTIP_CONTEXT_CTRL_LMB] = "Remove pen"
+
+	. = CONTEXTUAL_SCREENTIP_SET
```
Comment (Mothblocks) [inline code/modules/modular_computers/computers/item/tablet.dm:47]:
```suggestion
	return CONTEXTUAL_SCREENTIP_SET
```

**Review by Mothblocks — CHANGES_REQUESTED** [review]
No `. =` on trailing lines

**Pull request merged**